Bio/GenBank/__init__.py

```diff
@@ -654,14 +662,14 @@ def size(self, content):
         """Record the sequence length."""
         self._expected_size = int(content)
 
-    def residue_type(self, type):
+    def residue_type(self, res_type):
```

Comment:
Ya, good point. Seeing as the builtin `type` isn't used in the function, I guess is doesn't matter. Revert? Keep?